PluginCore/ScintillaNet/ScintillaControl.cs

```diff
@@ -3190,10 +3190,9 @@ public void GotoPos(int pos)
         /// </summary>
         unsafe public string GetCurLine(int length)
         {
-            int sz = SPerform(2027, length, 0);
-            byte[] buffer = new byte[sz + 1];
+            byte[] buffer = new byte[length + 1];
             fixed (byte* b = buffer) SPerform(2027, length + 1, (uint)b);
-            return Encoding.GetEncoding(this.CodePage).GetString(buffer, 0, sz - 1);
+            return Encoding.GetEncoding(this.CodePage).GetString(buffer, 0, length);
```

Comment:
This could cause problems as the returned value is different now, I'm OK with the change but I think it would be better to do:

```
        unsafe public string GetCurLine(int length)
        {
            int sz = SPerform(2027, 0, 0);
            byte[] buffer = new byte[length + 1];
            fixed (byte* b = buffer) SPerform(2027, length + 1, (uint)b);
            return Encoding.GetEncoding(this.CodePage).GetString(buffer, 0, Math.Min(length, sz - 1));
        }
```

At least this way when we try to get more chars than available we won't get back null characters at the end. Which some plugins devs may not expect with high length values.